### PR TITLE
feat: added journal club md

### PR DESCRIPTION
**Pull request by sogolsahebi**
This is attempt for the pull request ,the md file is in progress.

### CODE REVIEW

**Comment by sogolsahebi** [comment]
The journal club is ready for review!

File: docs/General/Meetings/journal_club.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- **Apply**: Optionally consider incorporating the findings or methods into your projects.
+
+It’s all about **learning**, **critiquing**, and **getting inspired**.
+
+---
+
+## **When and Where?**
+
+- **Day**: Wednesdays  
+- **Time**: 1:00–2:00 PM
```
Comment (a2nabc) [inline docs/General/Meetings/journal_club.md:21]: The time is 12:00 right? Should change that :)

**Review by a2nabc — CHANGES_REQUESTED** [review]
There's only one small change but this page looks great ;)

File: docs/General/Meetings/journal_club.md (hunk 1 above)
Comment (sogolsahebi) [reply docs/General/Meetings/journal_club.md]: Thank you , it is fixed now

**Review by a2nabc — APPROVED** [review]
great ;)

**Pull request merged**